src/cluster_legacy.c

```diff
@@ -4847,7 +4866,7 @@ void clusterHandleManualFailover(void) {
     if (server.cluster->mf_primary_offset == replicationGetReplicaOffset()) {
         /* Our replication offset matches the primary replication offset
          * announced after clients were paused. We can start the failover. */
-        server.cluster->mf_can_start = 1;
+        manualFailoverCanStart();
```

Comment:
Should we rather invoke `resetManualFailover` and clean up `failover_auth_time` in that period?

We should anyway cleanup failover_auth_time in the resetManualFailover method.

Reply:
I have thought about putting it in resetManualFailover, but I was worried about introducing other problems since resetManualFailover is called in many places.

Reply:
@hpatro Do you have other concerns? i dont want to touch resetManualFailover.

Reply:
LGTM